src/AutoMapper/PropertyMap.cs

```diff
@@ -1,165 +1,154 @@
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Linq.Expressions;
-using System.Reflection;
-using AutoMapper.Internal;
-
-namespace AutoMapper
-{
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using AutoMapper.Internal;
+
+namespace AutoMapper
+{
     using static Internal.ExpressionFactory;
-    using static Expression;
-
-    [DebuggerDisplay("{DestinationMember.Name}")]
-    public class PropertyMap : IMemberMap
-    {
-        private readonly List<MemberInfo> _memberChain = new List<MemberInfo>();
-        private readonly List<ValueTransformerConfiguration> _valueTransformerConfigs = new List<ValueTransformerConfiguration>();
-
-        public PropertyMap(MemberInfo destinationMember, TypeMap typeMap)
-        {
-            TypeMap = typeMap;
-            DestinationMember = destinationMember;
-        }
-
-        public PropertyMap(PropertyMap inheritedMappedProperty, TypeMap typeMap)
-            : this(inheritedMappedProperty.DestinationMember, typeMap) => ApplyInheritedPropertyMap(inheritedMappedProperty);
-
+    using static Expression;
+
+    [DebuggerDisplay("{DestinationMember.Name}")]
+    public class PropertyMap : IMemberMap
```

Comment:
Can you please try fixing the diff somehow? It's probably GitHub's fault, but it does make things difficult.

Reply:
Seems that there have been a lot of whitespace changes, which here hidden in the IDE. Fix is coming soon, in the meantime, you can disable them on GitHub by setting the diff settings next to the review button.